pandas/tests/interchange/test_impl.py

```diff
@@ -392,3 +392,13 @@ def test_large_string():
     result = pd.api.interchange.from_dataframe(df.__dataframe__())
     expected = pd.DataFrame({"a": ["x"]}, dtype="object")
     tm.assert_frame_equal(result, expected)
+
+
+@pytest.mark.parametrize("dtype", ["Int8", "Int8[pyarrow]"])
+def test_nullable_integers(dtype: str) -> None:
+    # https://github.com/pandas-dev/pandas/issues/55069
+    pytest.importorskip("pyarrow")
```

Comment:
We only need to skip for the second case no? Any reason we don't just use `pytest.param("Int8[pyarrow], marks=td.skip_if_no("pyarrow"))`?

Reply:
because I didn't know you could do that 🙈